modules/core/shared/src/main/scala/lucuma/core/math/dimensional/unit.scala

```diff
@@ -104,10 +104,11 @@ trait GroupedUnitOfMeasure[UG, U] extends UnitOfMeasure[U] with GroupedUnitType[
 object GroupedUnitOfMeasure {
   def apply[UG, U](implicit unit: UnitOfMeasure[U]): GroupedUnitOfMeasure[UG, U] =
     new GroupedUnitOfMeasure[UG, U] {
-      val definition = unit.definition
+      val name = unit.name
+      val abbv = unit.abbv
     }
 
   // `UnitDefiniton`s are expected to be singletons.
   implicit def eqGroupedUnitOfMeasure[UG, U]: Eq[GroupedUnitOfMeasure[UG, U]] =
-    Eq.instance((a, b) => a.definition == b.definition)
+    Eq.fromUniversalEquals
```

Comment:
This will use the `equals` on `UnitOfMeasure` right?

Reply:
Yes. Which ends up matching on both `name` and `abbv`.